shell/platform/darwin/macos/framework/Source/FlutterBackingStoreData.mm

```diff
@@ -6,7 +6,7 @@
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterFrameBufferProvider.h"
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterIOSurfaceHolder.h"
 
-#include <OpenGL/gl.h>
+#import <OpenGL/gl.h>
```

Comment:
Nit: this should also be its own block between what is currently line 5 and line 6.